### PR TITLE
PATH PLANNER: clone lqrrt node into repo

### PR DESCRIPTION
**Pull request by ironmig**
I didn't like how we were handling our use of lqrRRT, where we just run the ROS demo in that repository (which may change and not be inline with NaviGator specific needs). It also requried us manually run a setup.py as lqRRT is not a catkin package. This addresses both of those problems, by cloning the current lqRRT ROS Demo into the repository so we can develop it in NaviGator in the future (or replace it with a different path planner).

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
-[submodule "gnc/lqRRT"]
-	path = gnc/lqRRT
-	url = https://github.com/jnez71/lqRRT
+[submodule "gnc/lqrrt"]
+	path = gnc/navigator_path_planner/lqrrt
```
Comment (RustyBamboo) [inline .gitmodules]: `path = gnc/navigator_path_planner/lqRRT`

Comment (RustyBamboo) [inline .gitmodules]: The path currently is `gnc/navigator_path_planner/lqRRT`

File: gnc/navigator_path_planner/package.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  <name>navigator_path_planner</name>
+  <version>0.0.0</version>
+  <description>Node for NaviGator's lqRRT based path planner</description>
+  <maintainer email="nez.jason7@gmail.com">jason</maintainer>
+  <license>MIT</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <depend>rospy</depend>
+  <depend>actionlib</depend>
+  <depend>actionlib_msgs</depend>
```
Comment (RustyBamboo) [inline gnc/navigator_path_planner/package.xml]: Need build_depend and run_depend for actionlib?

**Comment by RustyBamboo** [comment]
Replace lqrrt_ros.msg in `mission_control/navigator_alarm/navigator_alarm_handlers/station_hold.py`

**Comment by ironmig** [comment]
Good catch... I think I'm actually going to PR this to kinetic-upgrade because that's what everyone's working on

**Pull request merged**